src/ui/components/Header/UserOptions.tsx

```diff
@@ -53,25 +65,58 @@ function UserOptions({ setModal, noBrowserItem }: UserOptionsProps) {
           expanded={expanded}
           orientation="bottom"
         >
+          {viewMode === "dev" && ( // only show layout options in devtools
+            <>
+              <div className="row group">
+                <span className="flex-1 text-left">Layout</span>
+
+                <div className="flex space-x-1">
+                  <div onClick={() => onLayoutChange("ide")}>
+                    <Icon
+                      filename="dock-bottom-right"
+                      className={`${styles.icon} ${toolboxLayout === "ide" ? styles.selected : ""}`}
+                    />
+                  </div>
+
+                  <div onClick={() => onLayoutChange("left")}>
+                    <Icon
+                      filename="dock-left"
+                      className={`${styles.icon} ${
+                        toolboxLayout === "left" ? styles.selected : ""
+                      }`}
+                    />
+                  </div>
+
+                  <div onClick={() => onLayoutChange("bottom")}>
+                    <Icon
+                      filename="dock-bottom"
+                      className={`${styles.icon} ${
+                        toolboxLayout === "bottom" ? styles.selected : ""
+                      }`}
+                    />
+                  </div>
+                </div>
+              </div>
+
+              <div className={styles.divider}></div>
+            </>
+          )}
+
+          <button className="row group" onClick={onSettingsClick}>
+            <span className="flex-1 text-left">Settings</span>
+            <Icon filename="settings" className="bg-iconColor" />
+          </button>
+
           <button className="row group" onClick={onDocsClick}>
+            <span className="flex-1 text-left">Docs</span>
             <Icon filename="docs" className="bg-iconColor" />
-            <span>Docs</span>
           </button>
           <ExternalLink className="row group" href="https://discord.gg/n2dTK6kcRX">
+            <span className="flex-1 text-left">Chat with us</span>
             <Icon filename="help" className="bg-iconColor" />
-            <span>Chat with us</span>
           </ExternalLink>
-          <button className="row group" onClick={onSettingsClick}>
-            <Icon filename="settings" className="bg-iconColor" />
-            <span>Settings</span>
-          </button>
-          {window.__IS_RECORD_REPLAY_RUNTIME__ || noBrowserItem ? null : (
-            <button className="row group" onClick={onLaunchClick}>
```

Comment:
The `onLaunchClick` event handler should be deleted (above) since it's no longer being referenced.